### PR TITLE
Fix gitignore and Implement GUI menu

### PR DESCRIPTION
**Pull request by Asafb26**

### CODE REVIEW

File: Poker.AT/.gitignore [hunk 1]
```diff
@@ -0,0 +1,252 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (idanizi) [inline Poker.AT/.gitignore]: *לא היה כלום, כי אין כלום*
image

File: Poker.AT/ASAFF/ASAFF.csproj [hunk 2]
```diff
@@ -1,129 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (idanizi) [inline Poker.AT/ASAFF/ASAFF.csproj]: i just noticed it now  - all the AT project is inside of `ASAFF/` folder? why? ❓

File: Poker.AT/App1aaaaa/App.xaml.cs [hunk 3]
```diff
@@ -1,100 +0,0 @@
-﻿using System;
```
Comment (idanizi) [inline Poker.AT/App1aaaaa/App.xaml.cs]: ❓ ❓ ❓  what is this mess? 👎 
image

File: Poker.FE/Assets/MainMenu/Scripts/MainMenu.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Attached to Main Camera
+/// </summary>
+public class MainMenu : MonoBehaviour {
+    public Texture backgroundTexture;
+
+    private void OnGUI()
```
Comment (idanizi) [inline Poker.FE/Assets/MainMenu/Scripts/MainMenu.cs]: nice background! 👍 
image
i'm trying to find the line you load it

File: Poker.AT/App1aaaaa/App.xaml.cs (hunk 3 above)
Comment (idanizi) [inline Poker.AT/App1aaaaa/App.xaml.cs]: ohhhhhh... now i get it. you clear it at your last commits. excellent 👍
image

File: Poker.FE/Assets/MainMenu/Textures/loading_animation.anim [hunk 5]
```diff
@@ -0,0 +1,193 @@
+%YAML 1.1
```
Comment (idanizi) [inline Poker.FE/Assets/MainMenu/Textures/loading_animation.anim]: you wrote this file by yourself or its auto-generated?

Comment (Asafb26) [inline Poker.FE/Assets/MainMenu/Textures/loading_animation.anim]: semi write it, i used a tool for know what's im doing here. this part is the progress bar animation between scene and the initial loading of game resources. took alot of time to implement 😫

Comment (idanizi) [inline Poker.FE/Assets/MainMenu/Textures/loading_animation.anim]: wow nice!

**Pull request merged**